onchain/cairo/launchpad/src/launchpad/locker/lock_manager.cairo

```diff
@@ -297,27 +272,27 @@ pub mod LockManager {
         }
 
         fn user_locks_length(self: @ContractState, user: ContractAddress) -> u32 {
-            let user_locks: Vec<felt252> = self.user_locks.read(user);
-            user_locks.len
+            let user_locks_length = self.user_locks_list.entry(user).len();
```

Comment:
This variable is never used, so gonna always return false. Please let's check if we simple add a Map of Contract address and u64 count indexer list. Or if there is a way to get a len array maybe